### ISSUE TITLE
relicense to GPL

### ISSUE THREAD

**Issue by larskuhtz** [larskuhtz/ghci-pretty#1]
ghci-pretty depends on the [hscolour](http://hackage.haskell.org/package/hscolour) package which is licensed under GPL. ghci-pretty should therefor be put under GPL, too.

**Comment by xsebek** [comment]
Well, it's only LGPL so it _may_ be fine to link to it:
- https://hackage.haskell.org/package/hscolour-1.24.4/src/LICENCE-LGPL

But I am too lazy to read through it and really understand it.

It would be nice to have this licensed under GPL too, though. :slightly_smiling_face:

**Comment by larskuhtz** [comment]
Yeah, if it is LGPL then it is probably fine. The question is if it is derived work. One could argue that the main functionality of the package comes from pretty printing, and syntax highlighting is just a generic add-on -- and there are alternatives packages that would provide the same functionality.

I am also fine with relicensing under GPL -- since it is a tool and not a library, it doesn't really matter and wouldn't break any user.
